packages/block-library/src/gallery/theme.scss

```diff
@@ -0,0 +1,4 @@
+.blocks-gallery-grid + figcaption,
+.wp-block-gallery > .block-editor-rich-text {
```

Comment:
I don't get these. :) Why target rich text? Why do we need styles for the gallery caption but not the image captions?

Reply:
Do you mean the image captions inside the gallery or the image block? Both of these have their own styles which are being applied by targeting the `figcaption` element. 
The problem is for the gallery block editor styles, there's no way of targeting the gallery `figcaption` without also including the gallery image `figcaptions`. This is another situation where using classnames instead of targeting elements directly would make our lives a lot easier, but it doesn't seem to be possible to pass a classname in to the rich text component, so I had to resort to this hacky workaround.

Reply:
OK I'm not sure what I was doing wrong before, but have now managed to add a classname to the caption so removing this hack.